kubernetes/resource_kubernetes_pod_test.go

```diff
@@ -883,13 +868,62 @@ func TestAccKubernetesPod_enableServiceLinks(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesPod_bug961EmptyBlocks(t *testing.T) {
+	name := acctest.RandomWithPrefix("tf-acc-test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckKubernetesPodDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesPodConfigEmptyBlocks(name, busyboxImageVersion),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttrSet("kubernetes_pod.test", "metadata.0.generation"),
+					resource.TestCheckResourceAttrSet("kubernetes_pod.test", "metadata.0.resource_version"),
+					resource.TestCheckResourceAttrSet("kubernetes_pod.test", "metadata.0.uid"),
+					resource.TestCheckResourceAttrSet("kubernetes_pod.test", "metadata.0.uid"),
+				),
+			},
+			{
+				Config:   testAccKubernetesPodConfigMinimal(name, busyboxImageVersion),
+				PlanOnly: true,
+			},
+		},
+	})
+}
```

Comment:
This test will error until we fix [bug 961](https://github.com/hashicorp/terraform-provider-kubernetes/issues/961) :sweat_smile: